client/src/components/Navbar/SearchBox.css

```diff
@@ -0,0 +1,57 @@
+.search-box-container {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: stretch;
+}
+
+form {
+	color: #555;
+	display: flex;
+	padding: 2px;
+	border: 1px solid gray;
+	border-radius: 5px;
+}
+
+input[type="search"] {
+	font-size: 15px;
+	padding: 7px 8px;
+	background: transparent;
+	border: none;
+	border: 1px solid transparent;
+}
+
+input[type="search"]::placeholder {
+	opacity: 0.4;
+}
+
+button[type="submit"] {
```

Comment:
can you use class to target these elements otherwise you will update all buttons that have type submit with this style 
for ex .example button[type="submit"]{

}